src/datadoc_editor/frontend/callbacks/dataset.py

```diff
@@ -114,6 +114,20 @@ def open_dataset_handling(
                 ),
                 dataset_opened_counter,
             )
+        failed_items = [
+            item["name"]
+            for item in getattr(state.metadata, "dataset_consistency_status", [])
```

Comment:
```suggestion
            for item in state.metadata.dataset_consistency_status or []
```

I don't think we need to use `getattr` here since we know the name of the attribute already. `getattr` is often used when we want to dynamically get an attribute based on some string value.